kernel_config

```diff
@@ -791,26 +791,26 @@ CONFIG_INET_UDP_DIAG=m
 CONFIG_TCP_CONG_CUBIC=y
 CONFIG_DEFAULT_TCP_CONG="cubic"
 # CONFIG_TCP_MD5SIG is not set
-CONFIG_IPV6=m
+CONFIG_IPV6=y
 CONFIG_IPV6_ROUTER_PREF=y
 # CONFIG_IPV6_ROUTE_INFO is not set
 # CONFIG_IPV6_OPTIMISTIC_DAD is not set
 CONFIG_INET6_AH=m
-CONFIG_INET6_ESP=m
+CONFIG_INET6_ESP=y
 CONFIG_INET6_IPCOMP=m
-CONFIG_IPV6_MIP6=m
+CONFIG_IPV6_MIP6=y
 # CONFIG_IPV6_ILA is not set
 CONFIG_INET6_XFRM_TUNNEL=m
 CONFIG_INET6_TUNNEL=m
 CONFIG_INET6_XFRM_MODE_TRANSPORT=m
 CONFIG_INET6_XFRM_MODE_TUNNEL=m
 CONFIG_INET6_XFRM_MODE_BEET=m
-CONFIG_INET6_XFRM_MODE_ROUTEOPTIMIZATION=m
+CONFIG_INET6_XFRM_MODE_ROUTEOPTIMIZATION=y
```

Comment:
Same story on this one -- Debian uses `=m` so I think we ought to also unless we've got a good reason not to.